src/patternfly/components/DataList/data-list.scss

```diff
@@ -2,15 +2,25 @@ $pf-data-list--breakpoint--mobile: 380px;
 
 .pf-c-data-list {
   --pf-c-data-list--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
+  --pf-c-data-list--BorderTopColor: var(--pf-global--Color--light-200);
+  --pf-c-data-list--BorderTopWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-data-list--mobile--BorderTopWidth: var(--pf-global--spacer--sm);
+  --pf-c-data-list--BorderBottomColor: var(--pf-global--Color--light-200);
+  --pf-c-data-list--BorderBottomWidth: var(--pf-global--BorderWidth--sm);
```

Comment:
The top/bottom border of expanded rows doesn't extend to the top and bottom of the component because of the border additions. I added a PR with one way to fix if you're interested - https://github.com/christiemolloy/patternfly-next/pull/10

Reply:
Agreed, you definitely don't want it to overlap on mobile, but on desktop, I think it looks better if the blue is flush with the top border (and the last expanded row is flush with the bottom border).

It's also worth noting that in this PR, the non-first rows blue border overlaps the border above it

<img width="165" alt="Screen Shot 2019-03-08 at 5 02 25 PM" src="https://user-images.githubusercontent.com/35148959/54060862-f7015a80-41c3-11e9-8e3a-c0559a5fa887.png">

And the case you brought up re: mobile is an issue there.

<img width="49" alt="Screen Shot 2019-03-08 at 5 02 50 PM" src="https://user-images.githubusercontent.com/35148959/54060881-0aacc100-41c4-11e9-8bc0-d8cdd26b584c.png">

I'm in favor of the blue border overlapping the borders on desktop (like they do now), and updating what's there now so they don't overlap on mobile. Or we could have them not overlap anywhere. It might be worth asking @mceledonia, too.